google/groups.go

```diff
@@ -111,14 +112,10 @@ func (g *GoogleGroups) ListUsers(desiredAttrs []string) ([]internal.Person, erro
 	var members []internal.Person
 
 	for _, nextMember := range membersList {
-		// Do not include Extra Managers or ExtraOwners in list to prevent inclusion in delete list
-		if isExtraManager, _ := internal.InArray(nextMember.Email, g.GroupSyncSet.ExtraManagers); isExtraManager {
-			continue
-		}
-		if isExtraOwner, _ := internal.InArray(nextMember.Email, g.GroupSyncSet.ExtraOwners); isExtraOwner {
-			continue
-		}
-		if isExtraMember, _ := internal.InArray(nextMember.Email, g.GroupSyncSet.ExtraMembers); isExtraMember {
+		// Do not include ExtraManager, ExtraOwners, or ExtraMember in list to prevent inclusion in delete list
+		if slices.Contains(g.GroupSyncSet.ExtraManagers, nextMember.Email) ||
+			slices.Contains(g.GroupSyncSet.ExtraOwners, nextMember.Email) ||
+			slices.Contains(g.GroupSyncSet.ExtraMembers, nextMember.Email) {
```

Comment:
Nice optimization.

Reply:
The new `slices` package is a nice addition to the standard library.